Running Move unit tests
[ FAIL    ] 0x00000000000000000000000000000001::M::fail
[ FAIL    ] 0x00000000000000000000000000000001::M::fail_with_code

Test failures:

Failures in 0x00000000000000000000000000000001::M:

┌── fail ──────
│ Test did not abort as expected
└──────────────────


┌── fail_with_code ──────
│ Test did not abort as expected
└──────────────────

Test result: FAILED. Total tests: 2; passed: 0; failed: 2
